docs-ref-services/security-keyvault-certificates-readme-pre.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Azure Key Vault Certificate client library for Java
-keywords: Azure, java, SDK, API, keyvault, azure-security-keyvault-certificates,
+title: 
```

Comment:
Missing title information. Do you know why this might be? @scbedd

Reply:
Yeah. Something went wrong with the metadata fill in the script, but it's super odd to me. Normally if it couldn't get the title, it also wouldn't update the version on the title, but it does here. Gonna need to do a local repro to figure out the oddness.

Reply:
Injected.